src/appleseed/renderer/kernel/lighting/directlightingintegrator.cpp

```diff
@@ -164,6 +164,8 @@ void DirectLightingIntegrator::compute_outgoing_radiance_light_sampling_low_vari
 
         sampling_context.split_in_place(3, m_light_sample_count);
 
+        size_t light_sample_count = 0;
```

Comment:
`light_sample_count` and `m_light_sample_count` are too close. Choose another name.

In fact, aren't they always equal?

Reply:
This comes from the solid angle sampling part of the work. Solid angle sampling can fail for some shapes and in that case, we don't count them as samples. For this PR, you can revert it to the original code.